packages/calypso-products/src/plans-list.tsx

```diff
@@ -1970,6 +1973,23 @@ const get2023EnterprisGrideDetails = (): IncompleteWPcomPlan => ( {
 	get2023PricingGridSignupStorageOptions: () => [],
 } );
 
+// Not a real plan. This is introduced as a pilot as part of pau2Xa-5rG-p2.
+const get2023AgencyPlanDetails = (): IncompleteWPcomPlan => ( {
+	...getDotcomPlanDetails(),
+	group: GROUP_WPCOM,
+	type: TYPE_AGENCY_BLUE_HOST,
+	getTitle: () => i18n.translate( 'Agency' ),
+	getAudience: () => i18n.translate( 'Best for agencies' ),
+	getPlanTagline: () =>
+		i18n.translate(
+			"Manage multiple WordPress sites with WP Cloud's powerful infrastructure and Bluehost's platform experience."
```

Comment:
```suggestion
			"Manage multiple WordPress sites with WP Cloud’s powerful infrastructure and Bluehost’s platform experience."
```

Used the smart quotes for consistency with our landing pages and the translation keys.